pkg/rule/skiprule.go

```diff
@@ -9,23 +9,41 @@ import (
 )
 
 var _ Rule = &SkipRule{}
+var _ Severity = &SkipRule{}
 
 // SkipRule is a Rule that always reports a predefined status.
 type SkipRule struct {
 	id            string
 	name          string
+	severity      SeverityLevel
 	justification string
 	status        Status
 }
 
+// Option is an interface that allows adding additional parameters to the SkipRule instances
+type Option func(*SkipRule)
```

Comment:
```suggestion
// SkipRuleOption is an interface that allows adding additional parameters to the SkipRule instances
type SkipRuleOption func(*SkipRule)
```
One more thing, since `SkipRule` is in the `rule` package it would be best we make a more specific `Option` type name